app/model/commands/CreateWorkflowAtomCommand.scala

```diff
@@ -0,0 +1,89 @@
+package model.commands
+
+import java.util.Date
+import java.util.UUID._
+
+import com.gu.atom.data.IDConflictError
+import com.gu.contentatom.thrift.atom.media.{Category => ThriftCategory, MediaAtom => ThriftMediaAtom, Metadata => ThriftMetadata}
+import com.gu.contentatom.thrift.{Atom => ThriftAtom, _}
+import com.gu.media.logging.Logging
+import com.gu.pandomainauth.model.{User => PandaUser}
+import data.DataStores
+import model.commands.CommandExceptions._
+import model.{ChangeRecord, MediaAtom, WorkflowMediaAtom}
+
+import scala.util.{Failure, Success}
+
+case class CreateWorkflowAtomCommand(workflowMediaAtom: WorkflowMediaAtom, override val stores: DataStores, user: PandaUser)
+
+  extends Command with Logging {
+
+  type T = MediaAtom
+
+  def process(): MediaAtom = {
+    val atomId = randomUUID().toString
```

Comment:
Indirectly related,  but I chatted with @tomrf1 and @shaundillon about this. We don't want the `id` to be random `UUID` but a human readable path registered through the [`path-manager`](https://github.com/guardian/path-manager), for instance for an explainer: `/atom/explainer/what-you-should-know-about-obama`.

I think @tomrf1 has started to work on extending `path-manager` to provide this.